libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-bar-filter-modal-opened-args.ts

```diff
@@ -0,0 +1,11 @@
+import { Observable } from 'rxjs';
+
+/**
+ * Arguments passed when a filter modal is opened.
+ */
+export interface SkyFilterBarFilterModalOpenedArgs {
+  /**
+   * An Observable representing data passed into the filter modal as additional context.
```

Comment:
```suggestion
   * An observable representing data that is passed to the filter modal as additional context.
```
The Angular docs don't treat "observable" as a proper noun, so I don't think we need to here.

Reply:
addressed in new file